packages/utils/src/get-section-from-schedules.js

```diff
@@ -0,0 +1,15 @@
+module.exports = async ({ content, siteId, load }) => {
+  const { sectionQuery } = content;
+  const query = {
+    status: 1,
+    'site.$id': siteId,
+    alias: { $ne: 'home' },
+  };
+  if (sectionQuery.length) {
+    const sectionsFromScheds = sectionQuery.filter(schedule => schedule.siteId === siteId);
+    const foundSections = await Promise.all(sectionsFromScheds.map(section => load('websiteSection', section.sectionId, {}, query)));
```

Comment:
The `projection` argument should not be an empty object, otherwise all fields will return. Add the `projection` argument to this util function and pass it to the `load` function

Reply:
I need all the fields as this is also being used to return the section itself here: https://github.com/parameter1/base-cms/pull/233/files/bf814341b5cb4a668917cb74833ab34425de5490#diff-466f539f3ec0e3fd7f2308c36f524ccf2ff92b8ef482f01cf0cf5d0eeb98e4faR488

Reply:
> Do we want it to just return the first section regardless of whether or not it is aliased as "home",

@Shinsina yes, you are correct... I forgot about the alias requirement. I'll adjust my comments where necessary

Reply:
@Shinsina the projection should be passed in by the callees, (see my comment on L487 of `resolvers/platform/content.js`

Reply:
Also, I removed the comments asking to change the `filter` to `find`